### PR TITLE
Features/submit data

### PR DESCRIPTION
**Pull request by Jacksonmcguire**
## Is this a fix or a feature?
Two Features.
## What is the change?
When the form is submitted, the data that was in the input fields will be saved into a new instance of an activity project.
If input fields are left blank, an error message will be shown
## Where should the reviewer start?
* Lines 30-End of Main.js file
* Error message labels in index.html file
## How should this be tested?
This should be tested by both submitting the form with data and without. 
When submitting the form, have the DEV Tools console open to see if a user activity is being instantiated.
For submitting the form without the data, simply look for an error message.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -12,7 +12,10 @@ Error: #EFB7EC */
...
 margin: 0;
 font-family: 'Montserrat', sans-serif;
-color: #FFF;
+}
+
+body {
+  color: #FFF;
 }
 
 /* ---------HEADER-------------- */
```
Comment (owenhallgren) [inline styles.css:21]: On line 15, is changing this color property going to affect our other text now that it is not applied to all?

File: main.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+categoryContainer.addEventListener("click", activateIcon);
+buttonStartActivity.addEventListener('click', submitForm);
+
+function toggle(element, className) {
+  element.classList.toggle(className);
+}
+
+function preventDefault(e){
```
Comment (elizhahn) [inline main.js]: As of now this function isn't being used, This should be removed.

**Review by owenhallgren — APPROVED** [review]

File: main.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  e.preventDefault();
+}
+
+function activateIcon() {
+  displayActivatedIcon(event);
+}
+
+function displayActivatedIcon(event) {
```
Comment (elizhahn) [inline main.js:24]: naming conventions for event are inconsisitent

File: main.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    if(categoryInputs[i].checked) {
+      iconDeactivated[i].classList.toggle("hidden");
+      iconActivated[i].classList.toggle('hidden');
+      currentIcon = categoryInputs[i];
+    };
+  };
+};
+
+
+function submitForm(e) {
```
Comment (elizhahn) [inline main.js]: example of naming conventions not matching

File: main.js (hunk 2 above)
Comment (owenhallgren) [reply main.js]: Good catch!

File: styles.css (hunk 1 above)
Comment (elizhahn) [reply styles.css:21]: I think we have color properties being added to the proper containers currently

**Review by owenhallgren — APPROVED** [review]

**Pull request merged**